### ISSUE TITLE
Separated Blocks mode

### ISSUE THREAD

**Issue by MajMcCloud** [albertobsd/keyhunt#228]
Hey together,

I got an idea for a new feature to combine with XPoint, Address and Rmd160 mode.

Lets call it for now "Separated Blocks mode", (a better name is desired)

As usual you are having your bit range. Additional to this you should be able with this mode to:

- 1. have a parameter to set, to split the bit range up into chunks (lets say 10 chunks), maybe -cs for chunk size
- 2. have a parameter to set, which acts as a time limit for each chunks , maybe -tl for timelimit
- 3. keep track of the status of chunks and switch every time to the next chunk, continue from the beginning after reaching the last one (would be good to save the progress of each chunk into a file)

So in conclusion with the -B parameter, when it has been finished, it will increase the propability quite far.

What are your thoughts?

Warm regards
Florian

**Comment by albertobsd** [comment]
I am about to finish the SaveProgress option.
